qconsolidatedialog.py

```diff
@@ -47,6 +47,11 @@ def __init__(self, iface):
 
         self.workThread = None
 
+        self.project_name_lbl = QLabel("Project name")
+        self.project_name_le = QLineEdit("project")
```

Comment:
Here I would like to use the current project name and provide a default only if the current project hasn't been created (saved) yet.